### PR TITLE
Create SECURITY.md

### LINKED ISSUE #460: [OTHER] Security feature
### What would you like to share or ask?

Hi everyone. I was looking through the Insights section and they pointed out that a Security standard was missing in [the Community Standards tab](https://github.com/rupali-codes/LinksHub/community). Should we add such as feature? 

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by raghav1030**
I have documented the security policy for the project. Please tell if there's any room improvement.

(I have added only one email id that I knew , but we can add more mail ids of the maintainers who can review the proposed vulnerabilities in this project)

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+The security policy will be regularly reviewed and updated to ensure that it remains effective and relevant. The project maintainers will evaluate the vulnerability disclosure process, update secure coding guidelines, and revise the response timeline as needed.
+## Disclosure Policy
+
+The project will follow a coordinated disclosure policy, which means that vulnerabilities will be disclosed publicly only after they have been remediated. The project may work with external website owners to coordinate disclosure of vulnerabilities that affect their websites.
+
+
+## Legal Disclaimer
+
+The security policy includes a legal disclaimer that limits the liability of the project maintainers and contributors for any security vulnerabilities or incidents that occur as a result of using the [Linkshub](https://linkshub.vercel.app/) website or any of the linked websites.
```
Comment (CBID2) [inline SECURITY.md:40]:
```suggestion
The security policy includes a legal disclaimer that limits the liability of the project maintainers and contributors for any security vulnerabilities or incidents that occur as a result of using[Linkshub's](https://linkshub.vercel.app/) website or any of the linked sources.
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+
+## Contact Information
+
+If you have any questions or concerns about the security policy or any security vulnerabilities in the project, please contact us at rupali7487@gmail.com.
+
+
+By implementing this security policy, we aim to ensure that vulnerabilities are addressed in a timely and responsible manner, and that users and contributors can use the [Linkshub](https://linkshub.vercel.app/) website and linked websites safely and securely.
```
Comment (CBID2) [inline SECURITY.md:48]:
```suggestion
By implementing this security policy, we aim to ensure that vulnerabilities are addressed in a timely manner, and that users and contributors can use [Linkshub](https://linkshub.vercel.app/) and its linked sources safely and securely.
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @raghav1030! 😊 Just made some suggestions. Great start 👍🏾

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+##  Secure Coding Practices
+
+The project will provide guidance on secure coding practices for contributors, including guidelines for input validation, authentication, authorization, and data protection
+## Regular Review and Update
+
+The security policy will be regularly reviewed and updated to ensure that it remains effective and relevant. The project maintainers will evaluate the vulnerability disclosure process, update secure coding guidelines, and revise the response timeline as needed.
+## Disclosure Policy
+
+The project will follow a coordinated disclosure policy, which means that vulnerabilities will be disclosed publicly only after they have been remediated. The project may work with external website owners to coordinate disclosure of vulnerabilities that affect their websites.
```
[the remark this answers is not in the document]
Comment (raghav1030) [reply SECURITY.md]: committed all changes

**Comment by raghav1030** [comment]
committed all the suggestion.
If anything more is to be done from my side then please tell?

**Review by CBID2 — APPROVED** [review]
LGTM! 😊

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+
+## Legal Disclaimer
+
+The security policy includes a legal disclaimer that limits the liability of the project maintainers and contributors for any security vulnerabilities or incidents that occur as a result of using[Linkshub's](https://linkshub.vercel.app/) website or any of the linked sources.
+
+
+## Contact Information
+
+If you have any questions or concerns about the security policy or any security vulnerabilities in the project, please contact us at rupali7487@gmail.com.
```
Comment (rupali-codes) [inline SECURITY.md:45]:
```suggestion
If you have any questions or concerns about the security policy or any security vulnerabilities in the project, please contact us at _linkshub.opensource@gmail.com_.
```

**Review by rupali-codes — COMMENTED** [review]
Please update the email address, rest looks great to me

**Comment by CBID2** [comment]
> Please update the email address, rest looks great to me

Hey @rupali-codes! 😄  I added your suggestion. I just need your approval so I can merge it

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**